### PR TITLE
ISSUE #4123 - UI Tweaks and Issues

### LINKED ISSUE #4123: UI Tweaks and Issues
A list of UI tweaks to improve the look and feel of the application overall

### Goals
- [ ] Improve the look and feel of the UI
- [ ] Make the UI more consistent

### Tasks
Tasks to be posted as individual comments inside this Issue

### PR DESCRIPTION
**Pull request by The-Daniel**
This fixes #4123 

#### Description
- Changed the border radius of
    - Dashboard lists
    - Bottom bar of viewer
    - all bordered inputs
    - buttons
    - alert messages
    - revisions list items
    - drag and drop boxes
    - empty boxes (basically all dashed containers)
    - tooltips
    - date and time popup
    - select dropdowns
- Changed the teamspace selection screen to [new design](https://www.figma.com/file/3FjNEOdQbcHwzquVZxUqMl/3D-Repo?node-id=3444%3A31963&t=X2Ge5J1OMVbC9nvt-1)  (also refactored link card)
- The notification button now just shows a colour dot when you have an unread notification. It no longer shows the number of notifications
- Removed the large buttons that appear at the bottom of the dashboard lists and jobs list
- Change search input styling to remove border and change background colour
- Remove grey borders from dashboard list

### CODE REVIEW

File: frontend/src/v4/routes/jobs/jobs.component.tsx [hunk 1]
```diff
@@ -32,7 +32,7 @@ import { NewJobForm } from '../components/newJobForm/newJobForm.component';
 import { UserManagementTab } from '../components/userManagementTab/userManagementTab.component';
 import { LoaderContainer } from '../userManagement/userManagement.styles';
 import { Panel } from '../components/floatingActionPanel/floatingActionPanel.styles';
-import { Container, NewJobBottomButton } from './jobs.styles';
+import { Container } from './jobs.styles';
 
 const JOBS_TABLE_CELLS = [{
 	name: 'Job name',
```
Comment (Amantini1997) [inline frontend/src/v4/routes/jobs/jobs.component.tsx:38]: At line 165, you can get rid of `openNewJobForm`. That was introduced for the bottom button and only used by that

**Review by Amantini1997 — CHANGES_REQUESTED** [review]

**Review by Amantini1997 — APPROVED** [review]

**Comment by george-hadfield** [comment]
### Add 5px border radius to Teamspace images

image

**Comment by george-hadfield** [comment]
### Add underline to 'Add new Teamspace'

image

**Comment by george-hadfield** [comment]
### Change number on accordion titles to be inside a circle
Change this containers also. Please add this functionality to project selection.

image

Link to design: https://www.figma.com/file/3FjNEOdQbcHwzquVZxUqMl/3D-Repo?node-id=3433%3A32163&t=1RzH7W1GKqCgiDhk-1

**Comment by Amantini1997** [comment]
I think we should also remove the border in user/project permissions
image

**Comment by george-hadfield** [comment]
### Change border-radius on logo container on Project Selection screen 

Change to 25px

image

**Comment by george-hadfield** [comment]
### Update font size below Teamspace title on Project selection screen
Please change to 12px body

image

**Comment by Amantini1997** [comment]
The border-radius missed the colour picker modal 
image

**Comment by george-hadfield** [comment]
### Set minimum width for Teamspace menu to 200px

Make the same for Federations and Containers

image

**Comment by george-hadfield** [comment]
### Change accordion buttons to rounded squares rather than circles

image

Link to design: https://www.figma.com/file/3FjNEOdQbcHwzquVZxUqMl/3D-Repo?node-id=3433%3A32163&t=1RzH7W1GKqCgiDhk-1

**Review by george-hadfield — CHANGES_REQUESTED** [review]
Quite a few comments and change

**Comment by george-hadfield** [comment]
### Make sure all the padding around the dashed empty states in the viewer are consistent

image

The custom tickets one is correct. The ones I can see are incorrect are:

- Issues
- Safetibase
- Views
- Tree
- Compare
- Sequences
- Measurements

**Comment by george-hadfield** [comment]
### Update bottom-toolbar to attached design

image

**Comment by george-hadfield** [comment]
### Ensure all button padding is correct

Incorrect example
image

Correct padding
https://www.figma.com/file/3FjNEOdQbcHwzquVZxUqMl/3D-Repo?node-id=3223%3A36635&t=1RzH7W1GKqCgiDhk-1

File: frontend/src/v5/ui/components/shared/linkCard/teamspaceCard/teamspacePlaceholderCard/teamspacePlaceholderCard.styles.ts [hunk 2]
```diff
@@ -16,31 +16,29 @@
...
+`;
+
+export const Card = styled(LinkCard)`
+	pointer-events: none;
+	${CardImage} {
+		${PlaceholderStyle}
+	}
+`;
+
+export const DisableHoverState = styled.div`
```
Comment (Amantini1997) [inline frontend/src/v5/ui/components/shared/linkCard/teamspaceCard/teamspacePlaceholderCard/teamspacePlaceholderCard.styles.ts:42]: If inside the `LinkCard` component you pass the `className` prop to the `CardListItem` component, you can get rid of `DisableHoverState`

**Review by Amantini1997 — CHANGES_REQUESTED** [review]

**Comment by Amantini1997** [comment]
I think you need to fix the storybook component for the link card image

**Comment by george-hadfield** [comment]
> ### Add underline to 'Add new Teamspace'
> image

Underline not showing

**Comment by george-hadfield** [comment]
> ### Change border-radius on logo container on Project Selection screen
> Change to 25px
> 
> image

Border radius still 10px

**Comment by The-Daniel** [comment]
image
image

I've noticed the search selects in custom tickets take on the new search styling. What should it do @george-hadfield ?

**Comment by Amantini1997** [comment]
I have only just noticed this: the notification sidebar gets the new border-radius
image

**Comment by Amantini1997** [comment]
The dropdown paper inside another paper does not get the styling 
image

**Comment by Amantini1997** [comment]
project permissions/user permissions tabs content is similar to the containers/federations list. Should they also get the outer border removed? 
image


> (The same goes for Jobs and Users, but the background around it is white, so I am just leaving this a note and not asking for changes )
image

**Comment by Amantini1997** [comment]
On hovering the items of the user menu, you can see that the border-radius is different from the button's 
image

Should they look alike?

**Comment by george-hadfield** [comment]
**I've noticed the search selects in custom tickets take on the new search styling. What should it do @george-hadfield ?**

It should NOT have the new styling. Can you please revert back to old styling?

**Comment by george-hadfield** [comment]
> On hovering the items of the user menu, you can see that the border-radius is different from the button's image
> 
> Should they look alike?

Yes please update

**Review by george-hadfield — CHANGES_REQUESTED** [review]
A few changes

**Comment by george-hadfield** [comment]
### Increase height of viewer inputs

Increase to 30px - currently 26px

**Comment by george-hadfield** [comment]
### Primary.main stroke and glow is now missing from the focused state of inputs
Can this please be restored?

image

**Comment by The-Daniel** [comment]
> ### Primary.main stroke and glow is now missing from the focused state of inputs
> Can this please be restored?
> 
> image

Do you mean on the search inputs? It looks okay on normal text inputs

**Comment by carmenfan** [comment]
@george-hadfield  @The-Daniel we should close this off asap - any new requests should go into the anew issue.

We need to focus on feature work for 5.4

**Review by george-hadfield — DISMISSED** [review]

**Comment by Amantini1997** [comment]
Something happened to the breadcrumbs elements which are no longer aligned

**Staging:**
image

**Current branch:**
image

File: frontend/src/v4/routes/components/foldableMenu/foldableMenu.style.ts [hunk 3]
```diff
@@ -77,7 +78,7 @@ export const Wrapper = styled.div<{ top: number, left?: boolean }>`
 	max-width: 400px;
 	width: 100%;
 	box-shadow: 1px 1px 3px 0 ${COLOR.BLACK_20};
-	border-radius: 0 2px 2px 0;
+	border-radius: ${isV5() ? '8px' : '0 2px 2px 0'};
```
Comment (Amantini1997) [inline frontend/src/v4/routes/components/foldableMenu/foldableMenu.style.ts:81]: Could you override this directly in v5 within `src\v5\ui\v4Adapter\dialogs\v4DialogsOverrides.styles.ts`?

**Comment by Amantini1997** [comment]
Should these 2 have the same border-radius? (top is 10 and bottom is 8)
> Feel free to move this to the next issue
image

**Review by Amantini1997 — CHANGES_REQUESTED** [review]

File: frontend/src/v4/routes/components/filterPanel/components/childMenu/childMenu.styles.ts [hunk 4]
```diff
@@ -31,7 +31,8 @@ export const Wrapper = styled.div`
 	max-width: 400px;
 	width: 100%;
 	box-shadow: 1px 1px 3px 0 ${COLOR.BLACK_20};
-	border-radius: ${isV5() ? '8px' : '0 2px 2px 0'};
+	/* border-radius: ${isV5() ? '8px' : '0 2px 2px 0'}; */
```
Comment (Amantini1997) [inline frontend/src/v4/routes/components/filterPanel/components/childMenu/childMenu.styles.ts:34]: I think this is a left over

**Review by Amantini1997 — CHANGES_REQUESTED** [review]

File: frontend/src/v4/routes/components/filterPanel/components/childMenu/childMenu.styles.ts [hunk 5]
```diff
@@ -17,6 +17,7 @@
 
 import styled from 'styled-components';
 
+import { isV5 } from '@/v4/helpers/isV5';
```
Comment (Amantini1997) [inline frontend/src/v4/routes/components/filterPanel/components/childMenu/childMenu.styles.ts:20]: I think this is a leftover

File: frontend/src/v4/routes/components/foldableMenu/foldableMenu.style.ts [hunk 6]
```diff
@@ -20,6 +20,7 @@ import styled from 'styled-components';
 import List from '@mui/material/List';
 import ListItemButton from '@mui/material/ListItemButton';
 
+import { isV5 } from '@/v4/helpers/isV5';
```
Comment (Amantini1997) [inline frontend/src/v4/routes/components/foldableMenu/foldableMenu.style.ts:23]: leftover?

File: frontend/src/v5/ui/components/shared/sso/microsoftButton.component.tsx [hunk 7]
```diff
@@ -31,7 +31,6 @@ export const MicrosoftButton = styled(Button).attrs({
 	width: fit-content;
 	font-weight: 500;
 	font-size: 12px;
-	border-radius: 0;
```
Comment (Amantini1997) [inline frontend/src/v5/ui/components/shared/sso/microsoftButton.component.tsx:34]: Removing the border-radius here means the MS button has rounded corners. I am not sure we should do this, did George specifically asked for this?

File: frontend/src/v5/ui/controls/breadcrumbs/breadcrumbs.styles.ts [hunk 8]
```diff
@@ -30,6 +30,7 @@ export const Container = styled(Breadcrumbs)`
 			overflow: hidden;
 			display: block;
 			white-space: nowrap;
+			height: 31px;
```
Comment (Amantini1997) [inline frontend/src/v5/ui/controls/breadcrumbs/breadcrumbs.styles.ts:33]: This fixes all the segments that are white, but no the highlighted one (the rightmost one) which still is a bit uplifted
image

**Review by Amantini1997 — CHANGES_REQUESTED** [review]

**Review by Amantini1997 — APPROVED** [review]

**Review by george-hadfield — APPROVED** [review]

**Pull request merged**